manifests/init.pp

```diff
@@ -205,11 +205,10 @@
     notify   => Service['foreman-proxy'],
   }
 
-  class { 'certs::katello':
+  class { 'foreman_proxy_content::bootstrap_rpm':
     hostname       => $rhsm_hostname,
     deployment_url => $rhsm_url,
```

Comment:
As suggested elsewhere, we can drop the parameter and keep it as a private class parameter. It can be adjusted via hiera if needed, but then it's not part of the public API.